src/current/_includes/v23.1/sidebar-data/feature-overview.json

```diff
@@ -0,0 +1,82 @@
+{
+    "title": "Feature Overview",
```

Comment:
Given the importance of the Quickstart and the changes we continue to make there, I'd prefer to minimize the number of variables we're changing at once. Let's put Get Started above Feature Overview in the nav.

Reply:
Done.